scripts/genCurrencies.mjs

```diff
@@ -20,54 +22,103 @@ import { getCurrencyData } from './getCurrencyData.mjs';
 // eslint-disable-next-line import/no-named-as-default-member
 const { format, resolveConfig } = prettier;
 
+const baseOutputDir = path.resolve(
+  url.fileURLToPath(new URL('.', import.meta.url)), // __dirname es module style
+  '../packages/currencies/src/iso4217/amendments/168'
```

Comment:
I think eventually we would even want this broken out into another layer of abstraction, so it will auto generate other amendments. But I think leaving that as another step is okay.